pkg/util/kubernetes/kubelet/kubelet.go

```diff
@@ -214,7 +214,7 @@ func (ku *KubeUtil) getLocalPodList(ctx context.Context) (*PodList, error) {
 
 	err = ku.addContainerResourcesData(ctx, pods.Items)
 	if err != nil {
-		log.Errorf("Error adding container resources data: %s", err)
+		log.Debugf("Error adding container resources data: %s", err)
```

Comment:
Please add a // TODO to say it will be converted back to `Errorf` when the underlying function is able to detect when the socket is available

Reply:
Done 👍